### PR TITLE
Flag failing tests

### PR DESCRIPTION
**Pull request by gregorjerse**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -35,7 +35,7 @@ extras =
     !docs: test
 passenv =
     # Pass environment variables controlling project's tests.
-    py{36,37,38}{,-storage-credentials},migrations: RESOLWE_* DOCKER_* DJANGO_TEST_PROCESSES TRAVIS LC_*
+    py{36,37,38}{,-storage-credentials},migrations: RESOLWE_* DOCKER_* DJANGO_TEST_PROCESSES TRAVIS LC_* GITHUB_*
```
Comment (dblenkus) [inline tox.ini:38]: Remove `TRAVIS`

File: resolwe/flow/tests/test_executors.py [hunk 2]
```diff
@@ -278,6 +281,9 @@ def test_null_executor(self):
 
     # TODO: Debug why the 'test-memory-resource-alloc' process doesn't end with and error on Travis
     @unittest.skipIf(os.environ.get("TRAVIS", "") == "true", "Fails on Travis CI")
+    @unittest.skipIf(
```
Comment (dblenkus) [inline resolwe/flow/tests/test_executors.py:284]: Remove the check for Travis above.

Comment (gregorjerse) [reply resolwe/flow/tests/test_executors.py:284]: OK.

File: tox.ini (hunk 1 above)
Comment (gregorjerse) [reply tox.ini:38]: OK.

**Review by dblenkus — APPROVED** [review]

**Pull request merged**